root/static/scripts/common/components/CDTocLink.js

```diff
@@ -15,10 +15,13 @@ type Props = {
   +cdToc: CDTocT,
   +content: string,
   +subPath?: string,
+  +anchorPath?: string,
 };
 
-const CDTocLink = ({cdToc, content, subPath}: Props): React.Element<'a'> => (
-  <a href={entityHref(cdToc, subPath)}>
+const CDTocLink = (
+  {cdToc, content, subPath, anchorPath}: Props
```

Comment:
Add missing comma using
```shell
git diff --name-only master '**.js' | xargs eslint --fix
```

```suggestion
  {cdToc, content, subPath, anchorPath}: Props,
```